docs/types.md

```diff
@@ -133,3 +133,90 @@ The following example applies 2em padding on top and bottom and 3em padding left
     "icon-padding": [2, 3]
 }
 ```
+
+
+
+## ProjectionTransition
+
+The `projectionTransition` is used to configure which projection to use for the map.
+
+There are currently two projections implemented.
+
+- `mercator` - [Web Mercator projection](https://en.wikipedia.org/wiki/Web_Mercator_projection)
+- `stereographic` - [Stereographic projection](https://en.wikipedia.org/wiki/Stereographic_projection)
+
+And the following [presets](#use-a-projection-preset)
+
+- `globe` - Globe projection
```

Comment:
Is this in the right place?

Reply:
It's in the `v8.projection.type.values` - I pull it from there and display it on the projection site as well

https://maplibre.org/maplibre-style-spec/projection/

But the types.md doesn't allow for that, so I just wrote it by hand there too.

Reply:
I get it now as there is only one preset at the moment. 
But since it is explained below, I would only link to it as you did and remove this part.

Reply:
ah, yes. remove it